README.md

```diff
@@ -39,3 +49,41 @@ end
 Another advantage of using `selenium` is that while running the specs, it opens s browser window and shows live the actions it performs, which could be useful for debugging the specs.
 
 The configuration for the drivers can be found in the `spec/rails_helper`.
+
+## Registration flow
+  As soon as an applicant sends their submission the following things happen:
+  1. If a submission breaks the default rules (an applicant took part in RailsGirls before or they can't speak English), it is rejected (see "Rejected tab"). Else, it shows in the "Valid" tab.
+  * Admins use "To rate" tab to assess the submission - show a submission, comment on it and give a rate.
+  * If the number of admins who rated a submission is large enough (see "Settings"), the submission is moved from "To rate" tab to "Results" tab.
+  * "Results" hold all rated submission ordered by their average rate. At first, they should have "Invitation" status of "not invited".
+  * You can send an email to the set number of applicants from the top of the list (to set the number see "Settings") - their "Invitation" status should change to "invited".
+  * The invited applicants have some time to confirm the invitations (to set the time see "Settings"). If the invitation is confirmed in the time, the "Invitation" status changes to "confirmed". Else, the status is changed to "expired".
+  * All the applicants who confirmed an invitation can be seen in "Participants" tab. It is your participants list, which you can download by clicking a "Download" button.
+
+## Setting automatic emails and reminders
+
+## Adapting the app for other events
+If you go through the process and have to change something that's not mentioned here, please create a PR or drop us a line!
```

Comment:
If you had to change something that is not mentioned here, while going through the process, please...